.github/workflows/Release.yaml

```diff
@@ -142,50 +142,14 @@ jobs:
               repo: context.repo.repo,
               ref: `heads/${context.payload.pull_request.head.ref}`,
             })
-  push_docker:
-    needs: release
-    name: Push Docker Image
-    runs-on: ubuntu-22.04
-    timeout-minutes: 720
-    strategy:
-      matrix:
-        rosdistro: [humble]
-        arch: [amd64]
-        # Build test for arm64 CPU is broken.
-        # This is a temporary solution and will be repaired in the future.
-        # See also https://github.com/tier4/scenario_simulator_v2/pull/1295
-        # arch: [amd64, arm64]
-    steps:
-      - name: Free Disk Space (Ubuntu)
-        uses: jlumbroso/free-disk-space@main
-        with:
-          tool-cache: false
-
-      - name: Install docker for ubuntu runner
-        uses: docker/setup-buildx-action@v3
-
-      - name: Install QEMU
-        uses: docker/setup-qemu-action@v3
-
-      - uses: actions/checkout@v3
 
-      - name: Login to GitHub Container Registry
-        uses: docker/login-action@v2
-        with:
-          registry: ghcr.io
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Build and push (${{ matrix.arch }})
-        uses: docker/bake-action@v3
-        with:
-          files: |
-            ./docker-bake.hcl
-          workdir: .
-          set: |
-            *.cache-to=type=gha,mode=max
-            *.cache-from=type=gha
-            *.tags=ghcr.io/tier4/scenario_simulator_v2:humble-${{ needs.release.outputs.new_version }}
-          push: ${{ github.event.pull_request.merged == true }}
-          targets: |
-            ${{ matrix.rosdistro }}_base_${{ matrix.arch }}
+      - name: Kick docker build action
+        if: github.event.pull_request.merged == true
+        run: |
+          curl -L \
+            -X POST \
+            -H "Accept: application/vnd.github+json" \
+            -H "Authorization: token ${{ secrets.BLOOM_GITHUB_TOKEN }}" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            https://api.github.com/repos/merge-queue-testing/scenario_simulator_v2/actions/workflows/Docker.yaml/dispatches \
+            -d '{"ref":"master","inputs":{"version":"3.5.2"}}'
```

Comment:
Is this for debug?
Please make it a variable, not a magic number, so it works in production.

Reply:
Sorry, it was my mistake.
I fix.

Reply:
This line was fixed.

Reply:
> **Please use production repository**  and use a variable, not a magic number, so it works in production.

There does not seem to be fixed about the bold part of the comment.